compute/admin_guide/authentication/oidc.adoc

```diff
@@ -116,12 +115,92 @@ Configure Prisma Cloud.
 
 .. Click *Save*.
 
+[.task]
+=== Azure Active Directory (AD)
+
+To integrate with Azure Active Directory (AD), you must register Prisma Cloud as an Open ID Connect (OIDC) application in Azure and configure Azure AD as an identity provider in Prisma Cloud.
+
+[.procedure]
+
+. Go to https://portal.azure.com/#home[your Azure console].
+
+. Find the Azure AD service.
+
+. Click the *app registration* button and select *New registration*
+
+. Enter a name and select *Twistlock only - single tenant* as the supported account type.
```

Comment:
That is based on Customer env (drop Twistlock only - single tenant). Can direct customers to choose - 'Accounts in this organizational directory only', but as mentioned that’s env dependent.

Reply:
What exactly do you mean with it being environment dependent?
What conditions need to be met for a customer to choose **Twistlock only - single tenant** over **Accounts in this organizational directory only**? 
Are there other options? Which is the most commonly used by our customers?
Aside from that, my question was also regarding the use of "Twistlock". I understand that may be how Azure has it listed but shouldn't it be instead "Prisma Cloud Compute"?
We'll document it as it appears in the UI for customers, but I mention it because we are trying to reduce the use of Twistlock in our docs in favor of Prisma Cloud Compute.

Reply:
'Twistlock' is our organizational directory name, and irrelevant in a customer env context.
Replace with - 'Accounts in this organizational directory only'